kosat-core/src/commonMain/kotlin/org/kosat/Clause.kt

```diff
@@ -21,6 +21,10 @@ data class Clause(
         return lits.map { it.toDimacs() }
     }
 
+    fun otherWatch(lit: Lit): Lit {
```

Comment:
I would prefer not to extract this as a method. Just calculate it when necessary and as necessary (xor'ing or if'ing), no need to complicate the API of a Clause.
The same goes for "`Lit::differentAmong2`" method.